src/mon/MgrMonitor.cc

```diff
@@ -54,6 +54,13 @@ void MgrMonitor::create_initial()
 	   << dendl;
 }
 
+void MgrMonitor::get_store_prefixes(std::set<string>& s)
+{
+  s.insert(service_name);
+  s.insert(command_descs_prefix);
```

Comment:
@jcsp @jecluis i am wondering if it'd be semantically more correct to reuse `service_name` as the prefix of the stored command description bl and use `$command_descs_prefix` as the key name here? if yes, i will file a ticket to address this as a follow-up of this fix.